src/hooks/useSettings.js

```diff
@@ -52,7 +52,7 @@ const useSettings = () => {
   const getConnectionSettings = (next) => {
     createNewRequest(
       espHttpURL("command", {
-        cmd: "[ESP800]",
+        cmd: "[ESP801]",
```

Comment:
As discussed in discord instead of creating new  command - I am in favor of an output flag for formating the output, backward compatibility is not the the target of 3.0, that is why the major number changed, I have added the flag :
`version=<webUIversion>` to [ESP800] is the FW want to support several version of webui

Reply:
As explained above, the version number is not helpful at all.  A flag is much better than a version number - although still not as good as a command name change for future changes.